src/components/index/blogPosts/PostCard.tsx

```diff
@@ -30,9 +32,15 @@ export default function ProjectCard({
                 </div>
               )}
             </div>
-            <div className="text-dark-1000 font-semibold px-3 mt-4 max-w-[294px] line-clamp-2 h-12 lg:text-lg lg:leading-6 lg:mt-6">
-              {post.title[0]?.text}
-            </div>
+            {post.title_de[0]?.text && router.locale === "de" ? (
```

Comment:
from your type in `src/components/commons/prismicTypes.ts`, isnt `title` and `title_de` an object? why are we doing array indexing here 🤔

Reply:
![image](https://user-images.githubusercontent.com/44110458/231953164-076c8230-ac12-432f-a625-4372fe274fee.png)
that's how the endpoint is returning the data (as seen in the screenshot). I have updated the `prismicTypes` so it is more accurate to the data given